packages/data/src/world/mm/overworld.yml

```diff
@@ -1564,7 +1563,7 @@
 "Near Deku Shrine":
   region: DEKU_PALACE
   exits:
-    "Deku Palace Front": "event(CLEAN_SWAMP) || has(MASK_DEKU)"
+    "Deku Palace Front": "event(CLEAN_SWAMP) || has(MASK_DEKU) || can_use_nayru"
```

Comment:
i wouldn't have this here unless when you die after being spit out by the big octo, you can use nayru before getting spit out again

Reply:
You can use it before the big octo gets you, and you can also get in a lilipad and use it.